python3/bin/perfmon

```diff
@@ -424,6 +424,7 @@ def get_percent_log_fs_usage(ignored):
     Get the percent usage of the host filesystem for logs partition.
     Input list is ignored and should be empty
     '''
+    _ = ignored  # unused: not sure if it'll be used later, passing pylint
```

Comment:
Could it be just like the following? Will the pylint complain this?
`def get_percent_log_fs_usage(_):`

Reply:
I tried and the pylint didn't complain about this.
Will updated the code.